tasks/config-plugin.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: GRAFANA | Download {{ item.name }}-{{ item.version }} plugin
```

Comment:
To be a party pooper 🤣 
This two task maybe could be one using
unarchive with option remote_src: true